src/views/Tenant/TenantView.jsx

```diff
@@ -55,6 +56,7 @@ const Tenant = () => {
     { id: "Closed", label: "Closed" },
   ];
   const [activeTab, setActiveTab] = useState(tabs[0].id);
+  const [ticketList, setTicketList] = useState();
```

Comment:
We should either add in a default to this useState() function (i.e. useState([])) or add in the ticketList to the initialState up above, initialized to an empty array